Apps/WebClient/src/ClientApp/app/components/timeline/laboratory.vue

```diff
@@ -239,6 +248,7 @@ export default class LaboratoryTimelineComponent extends Vue {
         link.download = `COVID_Result_${dateString}.pdf`;
         link.click();
         URL.revokeObjectURL(link.href);
```

Comment:
I think by this point the file would have already being downloaded? Should it instead happen on submit?